python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -799,6 +828,20 @@ def quantize(model: nn.Module,
             else:
                 invalidInputError(False,
                                   "Accelerator {} is invalid.".format(accelerator))
+        if precision == 'fp16':
+            invalidInputError(accelerator != 'openvino' or device not in ('GPU', 'VPUX'),
+                              "fp16 is not supported on {} device.".format(device))
```

Comment:
need to include accelerator in the error message too

Reply:
done